VoodooI2CELAN/VoodooI2CELANTouchpadDriver.cpp

```diff
@@ -29,6 +29,8 @@ bool VoodooI2CELANTouchpadDriver::check_ASUS_firmware(UInt8 productId, UInt8 ic_
 }
 
 bool VoodooI2CELANTouchpadDriver::init(OSDictionary *properties) {
+    OSSafeReleaseNULL(interrupt_simulator);
```

Comment:
Why are you release `interrupt_simulator` on driver initialisation? What if `interrupt_simulator` is not NULL, then what would happen when release is called? It's undefined behaviour. Best to remove it here OR set to NULL.

Reply:
Additionally, you should call `super::init`.

Reply:
fixed